### ISSUE TITLE
Wrong format for Date

### ISSUE THREAD

**Issue by xoewrl** [MaaxGr/intellij-jsonviewer#6]
```
before format 
{'time': '2023-05-04 20:05:59'} 

after format 
{
  'time': '2023-05-0420: 05: 59'
}
````

**Comment by MaaxGr** [comment]
wtf. Thanks for creating the issue. I will try to look into it soon. Please feel free to provide a pull request yourself, if you can find the exact problem

**Comment by MaaxGr** [comment]
Hey @doglex,
the Json Standard requires double quotes instead of single quotes.
https://stackoverflow.com/questions/36038454/parsing-string-as-json-with-single-quotes

Further i switched to kotlinx.serialization as a backend for formatting, so the issue should be fixed in 1.0.18.
Please reopen if you have any questions :)

Regards Max

**Comment by xoewrl** [comment]
> Hey @doglex, the Json Standard requires double quotes instead of single quotes. https://stackoverflow.com/questions/36038454/parsing-string-as-json-with-single-quotes
> 
> Further i switched to kotlinx.serialization as a backend for formatting, so the issue should be fixed in 1.0.18. Please reopen if you have any questions :)
> 
> Regards Max

ok
